carberretta/bot/cogs/meta.py

```diff
@@ -20,11 +20,92 @@
 
 import discord
 from discord.ext import commands
+from github import Github, UnknownObjectException, Issue
```

Comment:
Sort these imports alphabetically.

```suggestion
from github import Github, Issue, UnknownObjectException
```